src/main/java/org/zendesk/client/v2/model/Field.java

```diff
@@ -265,10 +265,10 @@ public String toString() {
         private String name;
         private String value;
 
-        private Option() {
+        public Option() {
```

Comment:
Why would you need to create an option with uninitialised values? This should only be required for Jackson and should be ok to leave as private.